### ISSUE TITLE
fontloader from the context minimals can no longer be used

### ISSUE THREAD

**Issue by u-fischer** [lualatex/luaotfload#385]
When I compile this minimal document in a current texlive 2016 and a current context minimals

```
\documentclass{article}
\usepackage{luaotfload}

\begin{document}
blub
\end{document}
```

with this entry in luaotfload.conf

```
  fontloader= context:D:/context-minimals/context/tex/texmf-context
```

I get an error about a nil-value in `font-tfm.lua`:

```
...ext/tex/texmf-context/tex/context/base/mkiv/font-tfm.lua:34: attempt to index local 'otf' (a nil value)
stack traceback:
    ...ext/tex/texmf-context/tex/context/base/mkiv/font-tfm.lua:34: in main chunk
    [C]: in function 'require'
    ...016/texmf-dist/tex/luatex/luaotfload/luaotfload-main.lua:198: in function 'l
oad_module'
    ...016/texmf-dist/tex/luatex/luaotfload/luaotfload-init.lua:307: in function 'l
oad_context_modules'
    ...016/texmf-dist/tex/luatex/luaotfload/luaotfload-init.lua:565: in function 'i
nit_main'
    ...016/texmf-dist/tex/luatex/luaotfload/luaotfload-init.lua:767: in function 'm
ain'
    ...016/texmf-dist/tex/luatex/luaotfload/luaotfload-main.lua:262: in function 'm
ain'
    [\directlua]:1: in main chunk.
l.49 }
```

**Comment by phi-gamma** [comment]
···<date: 2016-10-12, Wednesday>···<from: u-fischer>···

> When I compile this minimal document in a current texlive 2016 and a current context minimals
> 
> ```
> \documentclass{article}
> \usepackage{luaotfload}
> 
> \begin{document}
> blub
> \end{document}
> ```
> 
> with this entry in luaotfload.conf
> 
>       fontloader= context:D:/context-minimals/context/tex/texmf-context
> 
> I get an error about a nil-value in `font-tfm.lua`:
> 
> ```
> ...ext/tex/texmf-context/tex/context/base/mkiv/font-tfm.lua:34: attempt to index local 'otf' (a nil value)
> stack traceback:
> 	...ext/tex/texmf-context/tex/context/base/mkiv/font-tfm.lua:34: in main chunk
> 	[C]: in function 'require'
> 	...016/texmf-dist/tex/luatex/luaotfload/luaotfload-main.lua:198: in function 'l
> oad_module'
> 	...016/texmf-dist/tex/luatex/luaotfload/luaotfload-init.lua:307: in function 'l
> oad_context_modules'
> 	...016/texmf-dist/tex/luatex/luaotfload/luaotfload-init.lua:565: in function 'i
> nit_main'
> 	...016/texmf-dist/tex/luatex/luaotfload/luaotfload-init.lua:767: in function 'm
> ain'
> 	...016/texmf-dist/tex/luatex/luaotfload/luaotfload-main.lua:262: in function 'm
> ain'
> 	[\directlua]:1: in main chunk.
> l.49 }
> ```

What versions (luaotfload, Context) are you working with? There’s
been a pre-release of some sorts:

    https://github.com/lualatex/luaotfload/releases/tag/v2.8-rc2

My local working branch runs your example just fine with the
latest Context.

In any event, I have one urgent issue to fix then there will be
another pre-release so best wait for that one.

Best,
Philipp

**Comment by u-fischer** [comment]
I'm using normally using the luaotfload in the current texsystem(s) (miktex and TL16) and a current context minimals. With the prerelease (hurra for the tds.zip ...) the error is gone but I didn't test if it breaks something else.

**Comment by phi-gamma** [comment]
2016-10-20 (Thursday), notifications@github.com (u-fischer):

> I'm using normally using the luaotfload in the current
> texsystem(s) (miktex and TL16) and a current context minimals.

Ok.

> With the prerelease (hurra for the tds.zip ...) the error is gone
> but I didn't test if it breaks something else. 

Probably ;) Btw. there’s a new pre-release:
https://github.com/lualatex/luaotfload/releases/tag/v2.8-rc3

**Comment by u-fischer** [comment]
This has been resolved with luaotfload 2.9.
